tests/bgp/test_bgp_suppress_fib.py

```diff
@@ -124,14 +145,107 @@ def restore_bgp_suppress_fib(duthost):
     duthost.shell('sudo config save -y')
 
 
-@pytest.fixture(scope="module")
-def get_exabgp_ptf_ports(duthost, nbrhosts, tbinfo):
+@pytest.fixture(scope='module')
+def completeness_level(pytestconfig):
+    return pytestconfig.getoption("--completeness_level")
+
+
+@pytest.fixture(scope="function")
+def get_exabgp_ptf_ports(duthost, nbrhosts, tbinfo, completeness_level, request):
     """
     Get ipv4 and ipv6 Exabgp port and ptf receive port
     """
-    exabgp_port, ptf_recv_port = get_exabgp_port(duthost, nbrhosts, tbinfo, EXABGP_BASE_PORT)
-    exabgp_port_v6, ptf_recv_port_v6 = get_exabgp_port(duthost, nbrhosts, tbinfo, EXABGP_BASE_PORT_V6)
-    return exabgp_port, ptf_recv_port, exabgp_port_v6, ptf_recv_port_v6
+    is_random = True
+    if completeness_level == "thorough":
+        logger.info("Completeness Level is 'thorough', and script would do full verification over all VMs!")
```

Comment:
Can we make a parametrized regular test instead of an option?

Reply:
Yes, with the option approach it is required to run the BGP suppress test suite twice, one regular run and one with "--completeness_level=thorough", with the test being parametrized we don't need separate runs, it could save some time required for test setup.